DigitalLearningSolutions.Data.Tests/DataServices/TutorialContentDataServiceTests/TutorialTests.cs

```diff
@@ -42,5 +42,62 @@ public void GetTutorialIdsByCourse_returns_correct_tutorials()
             // Then
             result.Should().BeEquivalentTo(expectedTutorials);
         }
+
+        [Test]
+        public void UpdateOrInsertCustomisationTutorialStatuses_updates_both_statuses_on_existing_CustomisationTutorial()
+        {
+            // When
+            using var transaction = new TransactionScope();
+            try
+            {
+                tutorialContentDataService.UpdateOrInsertCustomisationTutorialStatuses(49, CustomisationId, false, false);
+                var result = tutorialContentDataService.GetTutorialsBySectionId(SectionId, CustomisationId).ToList();
+
+                using (new AssertionScope())
+                {
+                    result.First().TutorialId.Should().Be(49);
+                    result.First().TutorialName.Should().Be("View documents");
+                    result.First().Status.Should().BeFalse();
+                    result.First().DiagStatus.Should().BeFalse();
+                }
+            }
+            finally
+            {
+                transaction.Dispose();
+            }
+        }
+
+        [Test]
+        public void UpdateOrInsertCustomisationTutorialStatuses_inserts_new_CustomisationTutorial()
+        {
+            // Given
+            const int sectionId = 3059;
+            const int tutorialId = 12732;
+            const int customisationId = 14019;
+
+            // When
+            using var transaction = new TransactionScope();
+            try
+            {
+                var initialResult = tutorialContentDataService.GetTutorialsBySectionId(sectionId, customisationId)
+                    .ToList();
+                tutorialContentDataService.UpdateOrInsertCustomisationTutorialStatuses(tutorialId, customisationId, true, true);
+                var result = tutorialContentDataService.GetTutorialsBySectionId(sectionId, customisationId).ToList();
+
+                using (new AssertionScope())
+                {
+                    initialResult.First().Status.Should().BeNull();
+                    initialResult.First().DiagStatus.Should().BeNull();
+                    result.First().TutorialId.Should().Be(tutorialId);
+                    result.First().TutorialName.Should().Be("Create a presentation from an outline");

```

Comment:
I'm a bit confused here, should we also check that the TutorialId and Name for the initialResult are also correct, or shouldn't we be check the name/Id at all as it is somewhat irrelevant? Would this test be clearer if we had a new test helper method that just selects CustomisationTutorial records from a given tutorial and customisationId?